app/views/navbar.scala.html

```diff
@@ -11,9 +11,30 @@
 
 @import views.html.bootstrap._
 
-    <!-- Fixed navbar -->
+<!-- Fixed navbar -->
 <nav id="header" class="navbar navbar-default navbar-fixed-top">
     <div id="header-container" class="navbar-container container-fluid">
+        <!-- Banner is only visible if user is on test server-->
+        @if(Play.configuration.getString("environment-type").get == "test") {
+            <link href = "https://fonts.googleapis.com/icon?family=Material+Icons" rel = "stylesheet">
+            <div class = "test-server-banner">
+                <div class = "test-server-banner-content">
+                    <div class = "test-server-banner-text">
+                        <strong>WARNING!</strong> You are on the Test Server!
```

Comment:
Can you change this from "the Test Server" to "a test server"

Reply:
Done!